cmake/onnxruntime_session.cmake

```diff
@@ -39,3 +39,7 @@ endif()
 if (onnxruntime_ENABLE_TRAINING OR onnxruntime_ENABLE_TRAINING_OPS)
   target_include_directories(onnxruntime_session PRIVATE ${ORTTRAINING_ROOT})
 endif()
+
+if (onnxruntime_DEBUG_NODE_INPUTS_OUTPUTS)
+  target_compile_definitions(onnxruntime_session PRIVATE DEBUG_NODE_INPUTS_OUTPUTS)
```

Comment:
>DEBUG_NODE_INPUTS_OUTPUTS [](http://example.com/codeflow?start=57&length=25)

this is now used by session_state.h - it may be tricky to ensure it is consistently defined (or not) in all places that include session_state.h. perhaps the coarser-grained approach of defining this for the whole project would be simpler (e.g., in CMakeLists.txt with add_compile_definitions()).

Reply:
Yeah, agree, that's better. Let me change that.

Reply:
Moved to CMakeLists.txt.